design/button.behavior.md

```diff
@@ -0,0 +1,94 @@
+# Button Behavior
+
+Button behavior is the high level representation of how an MRE actor should behave as a button.  This behavior consists of two actions
+that occur on the object when it is targeted and when it is activated or clicked.  As part of these interactions, the point of targeting
```

Comment:
What is the difference between activated and clicked? You don't mention activate again, so maybe just remove it, or clarify that click is a way to activate the button.

Reply:
Activated versus clicked here is the same as button versus click below.  Updated the text here to reflect that.

---
In reply to: [428302504](https://github.com/microsoft/mixed-reality-extension-sdk/pull/593#discussion_r428302504) [](ancestors = 428302504)